modules/openapi-generator-core/src/main/java/org/openapitools/codegen/validation/ValidationRule.java

```diff
@@ -60,7 +60,7 @@ public String getFailureMessage() {
      *
      * @return <code>true</code> if the object state is valid according to this rule, otherwise <code>false</code>.
      */
-    public boolean evaluate(Object input) {
```

Comment:
Note this API change. I _think_ this is unused by users due to the package private APIs elsewhere in this PR. We can target 4.3 if there are concerns.

Reply:
I've rebased to 4.3 (planned for release in 5 weeks or so), just to be safe.